src/components/Nav.jsx

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import { Route, Routes, Link } from 'react-router-dom';
-import Books from './Books';
 import Categories from './Categories';
+import Ui from './Ui';
 
 function Nav() {
   return (
     <>
       <header className="head">
         <h1>Bookstore CMS</h1>
         <nav className="navigation">
-          <Link to="/" className="link">books</Link>
+          <Link to="Ui" className="link">books</Link>
           <Link to="/Categories" className="link">Categories</Link>
         </nav>
       </header>
       <Routes>
-        <Route path="/" element={<Books />} />
+        <Route path="Ui" element={<Ui />} />
```

Comment:
- [ ] When I open your app, nothing is displayed. You need to change this "Ui" to the root route, like so:

  ```
  <Route path="/" element={<Ui />} />
  ```